pkg/listener/mysql.go

```diff
@@ -780,8 +781,22 @@ func (l *MysqlListener) ExecuteCommand(ctx context.Context, c *mysql.Conn, data
 			l.stmts.Delete(stmtID)
 		}
 	case constant.ComStmtSendLongData: // no response
-		values := make([]byte, len(data))
-		copy(values, data)
+		if len(data) < 6 {
+			return err2.ErrMalformedPkt
+		}
+
+		stmtID := int(binary.LittleEndian.Uint32(data[0:4]))
+
+		si, ok := l.stmts.Load(stmtID)
+		if !ok {
+			return errors.Errorf("statement not found, statement id: %v", stmtID)
+		}
+		stmt := si.(*proto.Stmt)
+
+		paramID := int(binary.LittleEndian.Uint16(data[4:6]))
+		parameterID := fmt.Sprintf("v%d", paramID+1)
```

Comment:
why `v%d`, not `%d`?

Reply:
Same as packet/mysql.go
https://github.com/CECTC/dbpack/blob/be37d88f356e807d90fe57ffe17296752f65b7d5/pkg/packet/mysql.go#L99